packages/cx/src/widgets/form/Label.js

```diff
@@ -34,7 +34,9 @@ export class Label extends HtmlElement {
          context.parentDisabled
       );
 
-      if (instance.cache('disabled', data.disabled)) {
+      data.asterisk = context.parentAsterisk || this.asterisk;
+
+      if (instance.cache("disabled", data.disabled)) {
```

Comment:
If the value obtained from context changes, the label will not re-render. You can fix that with:

if (instance.cache("disabled", data.disabled) || instance.cache("asterisk", data.asterisk))